### ISSUE TITLE
Clarify Java Version?

### ISSUE THREAD

**Issue by aadrian** [tdf/odftoolkit#7]
Please clarify the Java version for this project.

The [Getting Started](https://github.com/tdf/odftoolkit#getting-started)  (and other seachable articles) still mention Java 6, but the `pom` has Java 8.

Thank you.

**Comment by svanteschubert** [comment]
We will be using Java 8 for the next Maven release and even this is EOL
(end-of-life for the Oracle JDKs).
I have not tested it with Java 11, but when I discussed the topic on the
FOSDEM conference, many other developers jumped from 8 to 11 to benefit
from the new features and tooling.
Someone interested in lending a helping and and test the Toolkit with Java
11?

As there are different opinions (some people prefer to stick with an old
JDK version) I do not want to rush it now, especially as my work on doing a
general overwork on the ODFDOM generation will not be finished before end
of year.

Cheers,
Svante

PS: Thanks for pointing out the mentionign of Java 6 in the readme, I fixed
it.

Am Di., 19. März 2019 um 09:58 Uhr schrieb aadrian <notifications@github.com
